File: problemas/iniciante/1184/1184.js
process.stdin.setEncoding('utf-8');

let input = '';

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    let entrada = input.trim().split('\n');
    let O = entrada.shift();
    
    let soma = 0.0;
    let count = 0;

    for (let i = 0; i < 12; ++i) {
        for (let j = 0; j < 12; ++j) {
            let valor = parseFloat(entrada.shift());

            if (i > j) {
                soma += valor;
                ++count;
            }
        }
    }

    if (O === 'S') {
        console.log(`${soma.toFixed(1)}`);
    }else if (O === 'M') {
        console.log(`${(soma / count).toFixed(1)}`);
    }

});

File: problemas/iniciante/1184/1184.py
def main():
    O = input()
    soma = 0.0
    count = 0

    for i in range(12):
        for j in range(12):
            valor = float(input())

            if i > j:
                soma += valor
                count += 1

    if O == "S":
        print(f"{soma:.1f}")
    else:
        print(f"{(soma / count):.1f}")

if __name__ == "__main__":
    main()
